copyright

```diff
@@ -100,6 +100,8 @@ License: CC-BY-SA-4.0
 Files:
  images/land/sky2.jpg
  images/land/sea3.jpg
+ images/land/beach4.jpg
+ images/land/canyon9.jpg
```

Comment:
As with everywhere else in the copyright file for images, you should drop the file extensions and instead have `*` at the end. And for the high DPI version, drop the `@2x` and the file extension for the `*`.